1.1/jessie/sdk/README.md

```diff
@@ -11,6 +11,8 @@ This repository contains images that are used to compile/publish ASP.NET Core ap
 
 - [`1.1.1`, `1.1`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/sdk/Dockerfile)
 - [`1.0.4`, `1.0`, `lts` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/jessie/sdk/Dockerfile)
+- [`1.1.1-nanoserver-10.0.14393.953` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/nanoserver/sdk/Dockerfile)
```

Comment:
To be consistent with microsoft/dotnet, you may consider only documenting the non-versioned variant (e.g 1.1.1-nanoserver).  Also to be consistent with debian, don't we want the other variants - 1.1-nanoserver, 1-nanoserver, nanoserver.